### PR TITLE
[Fleet] Managed Identity upgrade

### PR DESCRIPTION
**Pull request by JoyerJin**
## Description

Upgrade Fleet managed Identity design
Add example and test
## Mandatory Checklist

- Please choose the target release of Azure PowerShell
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Fleet/Fleet.Autorest/docs/Update-AzFleet.md [hunk 1]
```diff
@@ -8,38 +8,27 @@ schema: 2.0.0
 # Update-AzFleet
 
 ## SYNOPSIS
-Update a Fleet
+Create a Fleet.
```
Comment (BethanyZhou) [inline src/Fleet/Fleet.Autorest/docs/Update-AzFleet.md:11]: The verb should be Update. Needs a bug fix for autorest.

Comment (Nickcandy) [reply src/Fleet/Fleet.Autorest/docs/Update-AzFleet.md:11]: Can this pr be merged then ?

Comment (BethanyZhou) [reply src/Fleet/Fleet.Autorest/docs/Update-AzFleet.md:11]: Please hold on this PR until this synopsis is corrected.

**Review by BethanyZhou — APPROVED** [review]
LGTM

**Pull request merged**